query/statement_rewriter.go

```diff
@@ -69,25 +69,39 @@ func rewriteShowFieldKeyCardinalityStatement(stmt *influxql.ShowFieldKeyCardinal
 
 	return &influxql.SelectStatement{
 		Fields: []*influxql.Field{
-			{
+			&influxql.Field{
```

Comment:
I'm surprised we need the type name here - there's a lint we can turn on somewhere for that I think.